iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/VoltageAngleLimitXml.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package com.powsybl.iidm.xml;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.xml.XmlUtil;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.TerminalRef;
+import com.powsybl.iidm.network.VoltageAngleLimit;
+import com.powsybl.iidm.network.VoltageAngleLimitAdder;
+import com.powsybl.iidm.network.VoltageAngleLimit.FlowDirection;
+import com.powsybl.iidm.xml.util.IidmXmlUtil;
+
+import javax.xml.stream.XMLStreamException;
+
+/**
+ *
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public final class VoltageAngleLimitXml {
+
+    static final String VOLTAGE_ANGLE_LIMIT = "voltageAngleLimit";
+
+    static final String ID = "id";
+    static final String SIDE = "side";
+    static final String LIMIT = "limit";
+    static final String FLOW_DIRECTION = "flowDirection";
+    static final String FROM = "from";
+    static final String TO = "to";
+
+    public static void write(VoltageAngleLimit voltageAngleLimit, NetworkXmlWriterContext context) {
+        IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_10, context, () -> {
```

Comment:
Its is a bump to 1.11

Reply:
We know. It is temporary. Before changing the PR to ready for review, we will bump to the next xiidm version.